rascal-vscode-extension/src/RascalExtension.ts

```diff
@@ -183,3 +217,105 @@ function calculateRascalREPLMemory() {
     }
     return "-Xmx800M";
 }
+
+async function newRascalProject(dest: string, name: string) {
+        
+    const baseDest = URI.file(dest);
+    vsfs.createDirectory(baseDest);
+
+    const metaPath = URI.joinPath(baseDest, "META-INF");
+    await vsfs.createDirectory(metaPath);
+
+    const srcPath = URI.joinPath(baseDest, "src/main/rascal/");
+    await vsfs.createDirectory(srcPath);
+
+    const rascalMFPath = URI.joinPath(metaPath, "RASCAL.MF");
+    writeFileSync(rascalMFPath.fsPath, rascalMF(name));
```

Comment:
As discussed earlier, we should do `await vsfs.writeFile` (which takes a byte array as argument, so we have to produce the utf8 encoded variant of the string). But that both allows us to work with any virtual file system, and be a better VS Code neighbor.